dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerWithStructuresView.java

```diff
@@ -0,0 +1,27 @@
+package com.dotcms.rest.api.v1.container;
+
+import com.dotmarketing.beans.ContainerStructure;
+import com.dotmarketing.portlets.containers.model.Container;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.List;
+
+public class ContainerWithStructuresView {
```

Comment:
Please rename this class to `ContainerWithContentTypesView`, as `Structure` was the old term used to refer to what we now call Content Type. Also, we need Javadoc in here too.

Reply:
this feedback has been covered with the recent commit @jcastro-dotcms please have a look - thanks